cdap-common/src/main/java/co/cask/cdap/common/conf/CConfigurationUtil.java

```diff
@@ -33,4 +35,23 @@ public static void copyTxProperties(CConfiguration cConf, org.apache.hadoop.conf
       }
     }
   }
+
+  /**
+   * Asserts that the given CConfiguration has valid properties.
+   * @param cConf the CConfiguration object to check
+   * @throws IllegalArgumentException if the given cConf is invalid.
+   */
+  public static void assertCConfValidity(CConfiguration cConf) {
```

Comment:
Please just call it `verify`. The method is already inside CConfigurationUtil and is taking a CConfiguration as argument. No need to be that super verbose.